packages/babel-plugin-makepot/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Internal
+
+-   Lodash: Refactor away from `_.merge()` ([#48239](https://github.com/WordPress/gutenberg/pull/48239)).
```

Comment:
In a changelog message I would say that the `lodash` dependency has been removed from the package. It's something that's visible and useful for the package consumer, unlike the purely internal `merge` refactor.

Reply:
Good call. Expanded the CHANGELOG message in  https://github.com/WordPress/gutenberg/pull/48239/commits/291f545e96247dc8d568cb7cedc6e7a78692e5d1